repos/system_upgrade/el7toel8/actors/repositoriesmapping/libraries/library.py

```diff
@@ -1,69 +1,74 @@
-import csv
 import os
 
-from leapp.exceptions import StopActorExecution
-from leapp import reporting
+from leapp.exceptions import StopActorExecutionError
+from leapp.libraries.common import config
 from leapp.libraries.stdlib import api
 from leapp.models import RepositoriesMap, RepositoryMap
 from leapp.models.fields import ModelViolationError
 
+REPOMAP_FILE = '/etc/leapp/files/repomap.csv'
```

Comment:
You haven't used this constant anywhere. You can use it, instead of passing it from the actor.py.

Reply:
Aah. Yeah. That's artefact from early changes. I will rewrite it